src/renderer/agents/mulmo_vision_agent.ts

```
@@ -0,0 +1,63 @@
+import { type AgentFunctionInfo, type AgentFunction } from "graphai";
+import { mulmoScriptSchema } from "mulmocast/browser";
```

ESLint

[error] Remove this unused import of 'mulmoScriptSchema'.

(sonarjs/unused-import)